ignite/config/chain/base/config.go

```diff
@@ -10,128 +10,128 @@ import (
 
 // Account holds the options related to setting up Cosmos wallets.
 type Account struct {
-	Name          string   `yaml:"name"`
-	Coins         []string `yaml:"coins,omitempty"`
-	Mnemonic      string   `yaml:"mnemonic,omitempty"`
-	Address       string   `yaml:"address,omitempty"`
-	CoinType      string   `yaml:"cointype,omitempty"`
-	AccountNumber string   `yaml:"account_number,omitempty"`
-	AddressIndex  string   `yaml:"address_index,omitempty"`
+	Name          string   `yaml:"name" doc:"local name of a key pair associated with an account"`
+	Coins         []string `yaml:"coins,omitempty" doc:"list of token balances for the account"`
+	Mnemonic      string   `yaml:"mnemonic,omitempty" doc:"account mnemonic"`
+	Address       string   `yaml:"address,omitempty" doc:"account address"`
+	CoinType      string   `yaml:"cointype,omitempty" doc:"coin type number for HD derivation (default 118)"`
+	AccountNumber string   `yaml:"account_number,omitempty" doc:"account number for HD derivation (less than equal 2147483647)"`
+	AddressIndex  string   `yaml:"address_index,omitempty" doc:"address index number for HD derivation (less than equal 2147483647)"`
 }
 
 // Build holds build configs.
 type Build struct {
-	Main    string   `yaml:"main,omitempty"`
-	Binary  string   `yaml:"binary,omitempty"`
-	LDFlags []string `yaml:"ldflags,omitempty"`
-	Proto   Proto    `yaml:"proto"`
+	Main    string   `yaml:"main,omitempty" doc:"build main path"`
+	Binary  string   `yaml:"binary,omitempty" doc:"binary path"`
+	LDFlags []string `yaml:"ldflags,omitempty" doc:"custom build ld flags"`
+	Proto   Proto    `yaml:"proto" doc:"holds proto build configs"`
 }
 
 // Proto holds proto build configs.
 type Proto struct {
 	// Path is the relative path of where app's proto files are located at.
-	Path string `yaml:"path"`
+	Path string `yaml:"path" doc:"relative path of where app's proto files are located at"`
 
 	// ThirdPartyPath is the relative path of where the third party proto files are
```

Comment:
unrelated, we should remove this.

Reply:
Still used in some parts of the code, we need to open an issue to check and remove